pom.xml

```diff
@@ -250,7 +250,7 @@
     <!-- Fedora-"like" systems. This is currently only used for the netty-tcnative dependency -->
     <os.detection.classifierWithLikes>fedora</os.detection.classifierWithLikes>
     <tcnative.artifactId>netty-tcnative</tcnative.artifactId>
-    <tcnative.version>2.0.20.Final</tcnative.version>
+    <tcnative.version>2.0.21.Final-SNAPSHOT</tcnative.version>
```

Comment:
@normanmaurer Once I get https://github.com/netty/netty-tcnative/pull/421 merged in, I can update the test to point to the new released version.

Reply:
ok cool... I will do a release this week and then we can do this :)